File: presentation/src/main/java/com/everis/authspike/view/views/ContactsView.java
package com.everis.authspike.view.views;

import com.everis.domain.model.LocalContact;
import com.everis.domain.model.P2PUser;

import java.util.List;

/**
 * Created by everis on 3/05/18.
 */

public interface ContactsView extends BaseView {

    void displayBatchContacts(List<LocalContact> contacts);

    void setSync(P2PUser user);
}

File: presentation/src/main/java/com/everis/authspike/view/views/ContactsView.kt
package com.everis.authspike.view.views

import com.everis.domain.model.LocalContact
import com.everis.domain.model.P2PUser

/**
 * Created by everis on 3/05/18.
 */

interface ContactsView : BaseView {

    fun displayBatchContacts(contacts: List<LocalContact>)

    fun setSync(user: P2PUser)
}
